ruleset.xml

```diff
@@ -8,9 +8,24 @@
   </description>
   <rule ref="rulesets/java/basic.xml"/>
   <rule ref="rulesets/java/braces.xml"/>
+  <rule ref="rulesets/java/design.xml">
+    <exclude name="UseUtilityClass"/>
+    <exclude name="ConfusingTernary"/>
```

Comment:
up to human what's more readable imho :)

https://pmd.github.io/pmd-5.8.1/pmd-java/rules/java/design.html#ConfusingTernary